src/CloudGraphsDFG/services/CommonFunctions.jl

```diff
@@ -322,8 +322,8 @@ function _matchmergeVariableSubnode!(
         result = commit(tx)
     end
     length(result.errors) > 0 && error(string(result.errors))
-    length(result.results[1]["data"]) != 1 && error("Cannot find subnode '$(ppe.solveKey)' for variable '$variablekey'")
-    length(result.results[1]["data"][1]["row"]) != 1 && error("Cannot find subnode '$(ppe.solveKey)' for variable '$variablekey'")
+    length(result.results[1]["data"]) != 1 && error("Cannot find subnode '$(subnode)' for variable '$variablekey'")
+    length(result.results[1]["data"][1]["row"]) != 1 && error("Cannot find subnode '$(subnode)' for variable '$variablekey'")
```

Comment:
The issue does seem to be `ppe` object is not in here, likely a copy paste mistake.  Printing out the `$subnode` like this will work.  I'm not sure if we have to print out the whole thing, because it could be large.  So I'd also be happy to just do:

```suggestion
    length(result.results[1]["data"][1]["row"]) != 1 && error("Cannot find desired subnode::$(typeof(subnode)) for variable '$variablekey'")
```